contracts/lbp/LBPManager.sol

```diff
@@ -234,6 +240,47 @@ contract LBPManager {
         vault.joinPool(lbp.getPoolId(), address(this), address(this), request);
     }
 
+    /**
+     * @dev start LBP just before one block
+     * @notice it can be invoked by anyone only once
+     */
+    function startLbp() external {
+        uint256 startTime;
+        uint256 blockTime = 5 minutes;
+        bool isSwapEnabled = lbp.getSwapEnabled();
+        (startTime, , ) = lbp.getGradualWeightUpdateParams();
+        require(
+            !isSwapEnabled && state == TaskState.Waiting,
+            "LBPManager: swapping already active"
```

Comment:
if state is Ended, message would be wrong

Reply:
Should refer to "enabled" OR "disabled", not "active" (state terminology is internal to the contract)

Reply:
I don't think we should revert if !isSwapEnabled. Should just set the state to Started and return.

Reply:
I have implemented the changes you requested.